libcontainer/cgroups/fs/memory.go

```diff
@@ -291,7 +291,9 @@ func getPageUsageByNUMA(cgroupPath string) (cgroups.PageUsageByNUMA, error) {
 	stats := cgroups.PageUsageByNUMA{}
 
 	file, err := os.Open(path.Join(cgroupPath, cgroupMemoryPagesByNuma))
-	if err != nil {
+	if os.IsNotExist(err) {
+		return stats, nil
+	} else if err != nil {
```

Comment:
`else` is redundant here because there's a `return` above it, but it's fine I guess.